docs/admin/cold-starts.md

```diff
@@ -0,0 +1,14 @@
+# Improve cold starts 
+
```

Comment:
suggestion to add (or leave out) 


## What is a cold start

A cold start is when a container isn't readily available due to lack of activity, and Cloud Run has to spin up a new one. Spinning up the container will take longer than a pre-warmed idle container.